README.md

```diff
@@ -7,7 +7,7 @@
 
 ## Introduction
 
-An MSBuild SDK that is capable of producing a SQL Server Data-Tier Application package (.dacpac) from a set of SQL scripts that can be subsequently deployed using the `Microsoft.SqlPackage` [dotnet tool](https://www.nuget.org/packages/Microsoft.SqlPackage). It provides much of the same functionality as the SQL Server Data Tools .sqlproj project format, but is built on top of the new SDK-style projects that were first introduced in Visual Studio 2017.
```

Comment:
Nit: in my opinion, "an" is correct instead of "a". I would pronounce "MSBuild" as "emm-ess-build", which would make "an" correct here. 

I know this is a topic of debate among English-speaking technologists, even more so around the term "SQL"! So I'm okay with either but thought I'd add my opinion